funnel/views/project.py

```diff
@@ -452,6 +453,8 @@ def edit(self) -> ReturnView:
         form = ProjectForm(obj=self.obj, account=self.obj.account, model=Project)
         if form.validate_on_submit():
             form.populate_obj(self.obj)
+            # trigger signal to generate thumbnail image
```

Comment:
That's not the reason for this signal, so the comment is incorrect. Best to remove. It's a _generic_ data change signal that's being used elsewhere to generate a thumbnail, so the comment should be on the receiving side.